### PR TITLE
Added mish layer doc opset

### PR DESCRIPTION
**Pull request by iimironov**

### CODE REVIEW

File: docs/ops/activation/Mish.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+**Attributes**: operations has no attributes.
+
+**Inputs**:
+
+*   **1**: Input tensor *x* of any floating point type. Required.
+
+**Outputs**:
+
+*   **1**: Result of computes Mish activation. Floating point tensor with shape and type matching the input tensor. Required.
```
Comment (lazarevevgeny) [inline docs/ops/activation/Mish.md:19]: Please, add section about supported input/output data types.

File: docs/ops/activation/Mish.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+**Examples**
+
+```xml
+<layer ... type="Mish">
+    <input>
+        <port id="0">
+            <dim>256</dim>
+            <dim>56</dim>
+        </port>
+        <port id="1"/>
```
Comment (lazarevevgeny) [inline docs/ops/activation/Mish.md:38]: Why there are 3 input ports in this operation?

File: docs/ops/activation/Mish.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+## Mish <a name="Mish"></a>
+
+**Versioned name**: *Mish-4*
+
+**Category**: *Activation*
+
+**Short description**: Mish a Self Regularized Non-Monotonic Neural Activation Function.
+
+**Detailed description**: [Reference](https://www.tensorflow.org/addons/api_docs/python/tfa/activations/mish)
```
Comment (lazarevevgeny) [inline docs/ops/activation/Mish.md:9]: Please, do not provide the reference as a description. Instead, provide the textual description.

**Comment by lazarevevgeny** [comment]
Please, include other plugin owners as a code reviewers.

File: docs/ops/activation/Mish.md (hunk 1 above)
Comment (lazarevevgeny) [reply docs/ops/activation/Mish.md:19]: Again. Why the comment was resolved without actually changing the code?

Comment (lazarevevgeny) [inline docs/ops/activation/Mish.md:19]: Please, remove the sentence "Result of computes Mish activation."

File: docs/ops/activation/Mish.md (hunk 2 above)
Comment (lazarevevgeny) [reply docs/ops/activation/Mish.md:38]: I don't see code changes again.

File: docs/ops/activation/Mish.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+**Types**
+
+* *T*: any floating point type.
+
+**Mathematical Formulation**
+
+   For each element from the input tensor calculates corresponding
+    element in the output tensor with the following formula:
+    \f[
+    Mish( x ) = x*tanh{log{1+e^{x}}}
```
Comment (lazarevevgeny) [inline docs/ops/activation/Mish.md:30]: `log` -> `ln`

File: docs/ops/activation/Mish.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+## Mish <a name="Mish"></a>
+
+**Versioned name**: *Mish-4*
+
+**Category**: *Activation*
+
+**Short description**: Mish a Self Regularized Non-Monotonic Neural Activation Function.
```
Comment (vladimir-paramuzov) [inline docs/ops/activation/Mish.md:7]: Mish is ...

File: docs/ops/activation/Mish.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+**Detailed description**: Mish layer computes self regularized non-monotonic neural activation function: x * tanh(softplus(x))
```
Comment (lazarevevgeny) [inline docs/ops/activation/Mish.md:9]: Please, add a link to arxiv.

**Comment by nikita-kud** [comment]
I was looking through the specs for comparison and found a small format error in [GELU](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/activation/GELU_2.md):  
`Similarly, the following Gelu approximation (typical for the TensorFlow*) is recognized and fused into single Gelu op \f[ Gelu(x) \approx 0.5x(1 + tanh((sqrt(2/pi)) * (x + 0.044715 * x ^ 3)) \f] Inputs:`
Could you add new line for **Inputs:**?

File: docs/ops/activation/Mish.md [hunk 7]
```diff
@@ -4,9 +4,9 @@
 
 **Category**: *Activation*
 
-**Short description**: Mish a Self Regularized Non-Monotonic Neural Activation Function.
+**Short description**: Mish is a Self Regularized Non-Monotonic Neural Activation Function.
 
-**Detailed description**: Mish layer computes self regularized non-monotonic neural activation function: x * tanh(softplus(x))
+**Detailed description**: Mish layer computes self regularized non-monotonic neural activation function: x * tanh(softplus(x)) [Reference](https://arxiv.org/abs/1908.08681)
```
Comment (lazarevevgeny) [inline docs/ops/activation/Mish.md:9]: Please, remove the formula in this line and change the text to:
`...non-monotonic neural activation function proposed in the [article](https://arxiv.org/abs/1908.08681).`

**Comment by lazarevevgeny** [comment]
> Maybe it’s worth adding a link to [TF](https://www.tensorflow.org/addons/api_docs/python/tfa/activations/mish) in `Short description` as in the `GELU` spec?

I would not recommend doing this. In this particular case the activation function is simple and there is no need to reference other ML frameworks specification. I think that it is really enough to have a mathematical formula.

File: docs/ops/activation/GELU_2.md [hunk 8]
```diff
@@ -18,10 +18,12 @@ The following equivalent combination is recognized and fused into single Gelu op
     Gelu(x) = 0.5*x*(1 + erf((x) / sqrt(2) )
 \f]
 
-Similarly, the following Gelu approximation (typical for the TensorFlow*) is  recognized and fused into single Gelu op
+Similarly, the following Gelu approximation (typical for the TensorFlow*) is recognized and fused into single Gelu op 
+
 \f[
-    Gelu(x) \approx  0.5*x*(1 + tanh((sqrt(2/pi)) * (x + 0.044715 * x ^ 3))
+    Gelu(x) \approx 0.5x(1 + tanh((sqrt(2/pi)) * (x + 0.044715 * x ^ 3))
```
Comment (lazarevevgeny) [inline docs/ops/activation/GELU_2.md:24]: Change formula to 
`Gelu(x) \approx 0.5x(1 + tanh(\sqrt{2.0/pi} * (x + 0.044715 * x ^ 3))`

File: docs/ops/activation/Mish.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+**Types**
+
+* *T*: any floating point type.
+
+**Mathematical Formulation**
+
+   For each element from the input tensor calculates corresponding
+    element in the output tensor with the following formula:
+    \f[
+    Mish( x ) = x*tanh{ln{1+e^{x}}}
```
Comment (lazarevevgeny) [inline docs/ops/activation/Mish.md:30]: Change the formula to:
`Mish(x) = x*tanh(ln(1.0+e^{x}))`

File: docs/ops/activation/Mish.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
+## Mish <a name="Mish"></a>
```
Comment (lazarevevgeny) [inline docs/ops/activation/Mish.md:1]: @iimironov , please, update the docs/ops/opset4.md document.

File: docs/ops/activation/Mish_1.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
+## Mish <a name="Mish"></a>
+
+**Versioned name**: *Mish-1*
```
Comment (lazarevevgeny) [inline docs/ops/activation/Mish_1.md:3]: Change version to Mish-4

File: docs/ops/opset4.md [hunk 12]
```diff
@@ -27,6 +27,7 @@ declared in `namespace opset4`.
 * [CTCGreedyDecoder](sequence/CTCGreedyDecoder_1.md)
 * [Ceiling](arithmetic/Ceiling_1.md)
 * [Clamp](activation/Clamp_1.md)
+* [Mish](activation/Mish_1.md)
```
Comment (lazarevevgeny) [inline docs/ops/opset4.md:30]: Change version to Mish_4

Comment (lazarevevgeny) [reply docs/ops/opset4.md:30]: The operations must be sorted according to name. Why "Mish" is located between "Clamp" and "Concat"?

**Comment by rkazants** [comment]
@ilyachur , @GlebKazantaev , please merge this PR. We've got a few approvals and no concern's on yesterday's meeting. There should not be something critical in this specification. The operation is simple.

**Comment by ilyachur** [comment]
@rkazants Merged!

**Pull request merged**